go-selinux/selinux_linux.go

```diff
@@ -138,7 +138,7 @@ func verifySELinuxfsMount(mnt string) bool {
 		return false
 	}
 
-	if uint32(buf.Type) != uint32(unix.SELINUX_MAGIC) {
+	if uint32(buf.Type) != uint32(unix.SELINUX_MAGIC) { //nolint:gosec
```

Comment:
Can you use the `#nosec G115` style? That way we can supported only `G115` without any other gosec issue that may be found

Reply:
Alas this is not working for me. Perhaps you only used standalone `gosec`?

```console
$ golangci-lint run ./...
go-selinux/selinux_linux.go:141:11: G115: integer overflow conversion int64 -> uint32 (gosec)
	if uint32(buf.Type) != uint32(unix.SELINUX_MAGIC) { // #nosec G115 -- this code is right.
	         ^
```

Reply:
This is weird. The same annotation works just fine in a different place:

```go
i0 := int(c.TrailingZeroBits()) //#nosec G115 -- don't expect TralingZeroBits to return values with highest bit set.
```

bit not here. I am lost 😿

Reply:
OK, it doesn't work because it's after `{`. It works if I put in on a previous line:

```go
       //#nosec G115 -- there is no overflow here.
       if uint32(buf.Type) != uint32(unix.SELINUX_MAGIC) {
```

Reply:
But, in this case it might want for the whole block, rather than just this line. I'm lost again 😕

Reply:
Ah, the "whole block" is `return false` so I guess it's fine.

Reply:
Yeah, they are kinda horrible, the `#nosec` comments! I think it's good to use them though (to not accidentally mask other security-related discoveries).

Thanks!